lib/_http_outgoing.js

```diff
@@ -922,12 +912,13 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
 
   let ret;
   if (msg.chunkedEncoding && chunk.length !== 0) {
+    len ??= typeof chunk === 'string' ? Buffer.byteLength(chunk, encoding) : chunk.byteLength;
```

Comment:
(unrelated to this PR) As you mention here https://github.com/nodejs/node/issues/46603 we want to reduce this call by converting to buffer before reaching this stage?